### PR TITLE
Cleanup logger selection, enable ALL, NONE

### LINKED ISSUE #1108: Log levels ALL, NONE are not correctly parsed.
## Overview

**Bug report.**  SLF4J  supports logging levels ALL and NONE, but these are not correctly parsed by the parser in CorfuServer.

### PR DESCRIPTION
**Pull request by no2chem**
## Overview

Description: This PR cleans up the logger selection code in CorfuServer to use SLF4j's built in logger selection facility, which correctly parses ALL or NONE as potential settings.

Why should this be merged: This PR eliminates duplicate parsing of the log level parameter and enables two new log levels.

Related issue(s) (if applicable): Fixes #1108

## Checklist (Definition of Done):

- [x] There are no TODOs left in the code
- [x] [Coding conventions](https://github.com/CorfuDB/CorfuDB/wiki/Corfu-Style-Guidelines) (e.g. for logging, unit tests) have been followed
- [x] Change is covered by automated tests
- [x] Public API has Javadoc

### CODE REVIEW

**Comment by corfudb-performance** [comment]
Results automatically generated by _CorfuDB Benchmark Framework_ to assess the performance of this pull request for commit **352ef75**.

*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 1 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 5 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx+Scan workload, 10 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx workload, 1 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx workload, 5 threads, Disk mode
*** 0.0% transaction FAILURE rate for NonConflictingTx workload, 10 threads, Disk mode
 
 An interactive dashboard with Pull Request Performance Metrics for ALL cluster types and numbers of threads in run, is available at:
 [Pull Request #1109 Graphs ](https://snapshot.raintank.io/dashboard/snapshot/orBHIn8x3u80HwHphL9eCTdt0Eu4d913)

File: infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java [hunk 1]
```diff
@@ -210,28 +210,9 @@ public static void main(String[] args) {
...
-            case "TRACE":
-                root.setLevel(Level.TRACE);
-                break;
-            default:
-                root.setLevel(Level.INFO);
-                log.warn("Level {} not recognized, defaulting to level INFO",
-                        opts.get("--log-level"));
-        }
+        final Logger root = (Logger) LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME);
+        final Level level = Level.toLevel(((String)opts.get("--log-level")).toUpperCase());
```
Comment (medhavidhawan) [inline infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java]: What happens when the option is not there. The Docopts thing sets it to INFO ?

Comment (no2chem) [reply infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java]: Yes, same behavior as before

**Comment by no2chem** [comment]
@annym Thanks!

**Pull request merged**